### ISSUE TITLE
Left sidebar (admin-menu): match Calypso and wp-admin color schemes

### ISSUE THREAD

**Issue by fushar** [Automattic/wp-calypso#94471]
There seems to be discrepancies in certain admin color schemes between Calypso and wp-admin screens for the left sidebar.

Examples below are using **Classic Bright**:

---

### Site card (for Default admin interface style)

|Calypso|wp-admin|
|-|-|
|<img width="280" alt="image" src="https://github.com/user-attachments/assets/7f58dcc8-bc8e-476f-bc73-0546f8af5efd">|<img width="280" alt="image" src="https://github.com/user-attachments/assets/9d9002a4-e18c-4c8a-bc89-5a939ab2710c">|

Differences:

1. Site card borders: wp-admin's seems to have top and bottom borders.
2. Site card background also have different colors in some color schemes.
3. Coming soon badge background colors.
4. Margins around the "Free domain" upsell banner.

---

### Sidebar menus (both Default and Classic?)

|Calypso|wp-admin|
|-|-|
|<img width="280" alt="image" src="https://github.com/user-attachments/assets/840a9343-de88-4344-ae5f-3ee91cf33fc8">|<img width="280" alt="image" src="https://github.com/user-attachments/assets/cf11a8a0-a26b-4278-a4d2-472dfd03cca4">|

---

### Cleaning up "navredesign" variables

Can we now remove the `--color-navredesign-sidebar-menu-*` variables introduced from the following PRs?

- https://github.com/Automattic/wp-calypso/pull/90193
- https://github.com/Automattic/wp-calypso/pull/92397

They are confusing. IMO, we should have the same set of color definitions for both Default and Classic styles, and they need to match wp-admin's definitions.

**Comment by davemart-in** [comment]
Do we need to address this across all color schemes?

**Comment by fushar** [comment]
Yes, we need to audit all color schemes. I see that some already match, and some don't/

**Comment by ryelle** [comment]
The icons on the Calypso menus are also not uniform, items with children are a different color than the other icons:

| | wp-admin | Calypso |
|---|---|---|
| Aquatic | <img src="https://github.com/user-attachments/assets/9cb5b8a8-2e14-4548-9c20-e2fc4de1fc33" width="200" /> | <img src="https://github.com/user-attachments/assets/54955a3e-8dec-40d6-8f18-4dedb5fd32a3" width="200" /> |
| Classic Blue | <img src="https://github.com/user-attachments/assets/3a3e369e-7b6e-4437-b678-4b7d27976a8f" width="200" /> | <img src="https://github.com/user-attachments/assets/18065efd-342b-4808-8850-fa85923bb59c" width="200" /> |
| Light | <img src="https://github.com/user-attachments/assets/0170240c-ce15-4305-8642-1e4620f317a2" width="200" /> | <img src="https://github.com/user-attachments/assets/8e287a5c-522b-4d40-8776-0f0970c0ed38" width="200" /> |

Sakura's submenu color is also different between the two. In this case, I think the Calypso menu looks better, so the fix would need to happen wherever those colors are set ([here?](https://github.com/Automattic/jetpack/tree/trunk/projects/packages/masterbar/src/admin-color-schemes)).

| wp-admin | Calypso |
|---|---|
| <img src="https://github.com/user-attachments/assets/c083b1fa-d47e-44b2-bb5b-207a162c4e9a" width="200" /> | <img src="https://github.com/user-attachments/assets/fda38826-b409-4641-8447-5800dc5c5146" width="200" /> |
